codegen/projections/rails_json/lib/rails_json/middleware.rb

```diff
@@ -53,8 +53,10 @@ def self.build(config, options = {})
         stack.use(Middleware::RequestId)
         stack.use(Hearth::Middleware::Send,
           client: config.http_client,
+          response_events: false,
```

Comment:
Is there a true example here?

Reply:
Not for rails json - I don't think it really makes sense to try to add. 

But theres good examples in white label with this.